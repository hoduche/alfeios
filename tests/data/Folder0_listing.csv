Folder,File,Size
C:\netBeansProjects\fs-crawler\tests\data\Folder0,file3.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0,file5.txt,12
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder1,file3.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder1,file4.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder1,file5.txt,12
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder2,file3.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder2,file4.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder2,file5.txt,12
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder2\Folder2_1,file4.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder2\Folder2_1,file5.txt,12
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder2,maliciousFox.png,367623
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder2,scaredGroundhog.png,367623
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder2,Thumbs.db,164352
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder3,file3.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder3,file4.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder3,file5.txt,12
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder3\Folder3_1,file4.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder3\Folder3_1,file5.txt,12
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder3,fox.png,367623
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder3,groundhog.png,367623
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder3,Thumbs.db,164352
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder4,file3.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder4,file4.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder4,file5.txt,12
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder4\Folder4_1,file4.txt,5
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder4\Folder4_1,file5.txt,21
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder4,fox.png,367623
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder4,groundhog.png,367623
C:\netBeansProjects\fs-crawler\tests\data\Folder0\Folder4,Thumbs.db,164352
C:\netBeansProjects\fs-crawler\tests\data\Folder0,maliciousFox.png,367623
C:\netBeansProjects\fs-crawler\tests\data\Folder0,scaredGroundhog.png,367623
C:\netBeansProjects\fs-crawler\tests\data\Folder0,Thumbs.db,164352
