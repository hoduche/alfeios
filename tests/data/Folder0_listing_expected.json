{"('5d41402abc4b2a76b9719d911017c592', 'FILE', 5)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder1\\file3.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder1\\file4.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\Folder2_1\\file4.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\file3.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\file4.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\Folder3_1\\file4.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\file3.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\file4.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\Folder4_1\\file4.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\file3.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\file4.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\file3.txt"], "('6a4316b18e6162cf9fcfa435c8eb74c1', 'FILE', 12)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder1\\file5.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\Folder2_1\\file5.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\file5.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\Folder3_1\\file5.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\file5.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\file5.txt", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\file5.txt"], "('9f835231b3281b4283694082ef331982', 'DIR', 22)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder1"], "('9f835231b3281b4283694082ef331982', 'DIR', 17)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\Folder2_1", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\Folder3_1"], "('bb3752d6d543c87cbbe189d662d1fbe0', 'FILE', 367623)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\maliciousFox.png", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\scaredGroundhog.png", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\fox.png", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\groundhog.png", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\fox.png", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\groundhog.png", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\maliciousFox.png", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\scaredGroundhog.png"], "('ab959cc33b752f274820aff06e6c09ef', 'FILE', 164352)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2\\Thumbs.db", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3\\Thumbs.db", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\Thumbs.db"], "('b0b2d7be70aa7fede91c8fee9b21acb0', 'DIR', 899637)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder2", "C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder3"], "('47e22de6e32c293c04b137e039a8c3c8', 'FILE', 21)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\Folder4_1\\file5.txt"], "('057e154f3031934233ea262ad7aee051', 'DIR', 26)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4\\Folder4_1"], "('5bd69b32f8255cac058752ee08a4422a', 'DIR', 899646)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Folder4"], "('e340de8cf8fa0bc3f0d3f764a8a6b7f8', 'FILE', 164352)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0\\Thumbs.db"], "('f9f407c1723b0258989d1a69996e4ac1', 'DIR', 3598557)": ["C:\\netBeansProjects\\fs-crawler\\tests\\data\\Folder0"]}